### ISSUE TITLE
[Feature]: I would like to add a function because I found that the local database file cannot be loaded and viewed in certain paths.

### ISSUE THREAD

**Issue by WilliamGrant** [sqlitebrowser/sqlitebrowser#3820]
### Describe the new feature

I tried to load some local database files, but some of them failed. However, the software page did not show any error message, so I could not tell what the problem was. However, when I moved the file to the desktop or a shallow path, I could read the same database file. So I wondered if I could add a log function to view the errors that occurred in the software, or a prompt function. Here is the info what I failed to load:
system:deepin v23 beta
path:`/media/Mashiro/inst_47/electric-monitor/bin/data/tt-electric-monitor.db`

**Comment by justinclift** [comment]
Just to check, which version of the program are you using, and which OS is it on?

It *kind* of sounds like you might be using Linux?  If that's the case, are you using the Snap version of our application?  If you are, then the easiest workaround would probably be to use the .AppImage version of our application as that (in theory) shouldn't have any problems accessing mounted media: https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.13.1

**Comment by WilliamGrant** [comment]
Yes! This is indeed working on Linux, the software version is 3.12.2. I installed sqlite via apt. I will try to install the 3.13.1 version of the appimages later to see if the same problem occurs. Thanks for your advice!

**Comment by mgrojo** [comment]
Is there something in the "SQL Log" > "Error Log". The errors reported by SQLite go there, so you don't need to add a function to see them, as far as I understand.

Regarding the database that cannot be opened on certain paths, are those directories writable? See this https://www.sqlite.org/forum/forumpost/ecec426355
